kombu/transport/redis.py

```diff
@@ -942,7 +943,8 @@ def disconnect(self):
                     if self._sock is None:
                         return
                     try:
-                        # self._sock.shutdown(socket.SHUT_RDWR)
+                        if os.getpid() == self.pid:
```

Comment:
Please add a TODO to drop this once we drop py-redis < 3.2

Reply:
Done!